cli/dcoscli/node/main.py

```diff
@@ -4,18 +4,29 @@
     dcos node --info
     dcos node [--json]
     dcos node log [--follow --lines=N --master --slave=<slave-id>]
+    dcos node ssh [--option SSHOPT=VAL ...]
```

Comment:
Those are required.  These are optional.  I don't think we should have optional positional arguments.